.github/workflows/ci.yml

```diff
@@ -369,7 +369,8 @@ jobs:
       BAAS_BAASAAS_API_KEY: ${{ secrets.BAASAAS_API_KEY}}
     defaults:
       run:
-        working-directory: packages/realm/tests  
+        shell: bash
```

Comment:
Why is this necessary? It doesn't hurt, but bash is already the default shell on linux, isn't it?

Reply:
Yes, it is only needed for the generator tests, which also runs on windows. I'll revert here.